packages/cli-doctor/src/tools/runBundleInstall.ts

```diff
@@ -13,7 +13,6 @@ async function runBundleInstall(loader: Loader) {
     logger.error((error as any).stderr || (error as any).stdout);
     throw new CLIError(
```

Comment:
I'm curious: Why do we need to handle this on a case-by-case basis at the time of throwing the error? Is there *ever* a reason to print the stack trace of a `CLIError`?

Reply:
There are places where the stack trace would be handy. Following are examples to name a few :

https://github.com/react-native-community/cli/blob/main/packages/cli/src/index.ts#L183
https://github.com/react-native-community/cli/blob/main/packages/cli/src/commands/init/init.ts#L63
https://github.com/react-native-community/cli/blob/main/packages/cli-hermes/src/profileHermes/sourcemapUtils.ts#L27

Having stack trace emptied in case it's a human actionable error is what [CLIError](https://github.com/react-native-community/cli/blob/main/packages/cli-tools/src/errors.ts) is doing.